src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -172,10 +172,10 @@ const FormattedReport = ({ summaries, weekIndex, bioCanEdit }) => {
     }
   };
 
-  const BioSwitch = (userId, bioPosted, weeklySummaryOption, teamColor) => {
+  const BioSwitch = (userId, bioPosted, weeklySummaryOption, teamColor, totalTangibleHrs, daysInTeam) => {
     const [bioStatus, setBioStatus] = useState(bioPosted);
     return (
-      <div>
+      <div style={totalTangibleHrs > 80 && daysInTeam > 60 && bioPosted !== "posted" ? {backgroundColor: "yellow"}: {}}> 
```

Comment:
To improve code readability, please assign this expression to a `const` before the `return` and use it as the `style` value.

Reply:
@Cirakuze Hi, I addressed your comment, please review again. Thanks!

Reply:
@KaixiangGU, I am not able to access the Weekly Summary Report page. It loads for a while then blanks out. Have you come across this?

I'm on your FE branch and the latest BE dev branch. This happens in Chrome, Safari, and Firefox. I've tried reloading and restarting the app. I was using an Admin account. I am able to access the page on the latest FE dev branch.

Reply:
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/15364639/6bb69aca-ae26-4373-ac1c-f12516cefb2b

Reply:
Hi @Cirakuze I fixed issue, please review it again, thank you!